tests/data/SimpleWithExamplesCest.php

```diff
@@ -17,4 +17,11 @@ public function helloWorld(\CodeGuy $I, \Codeception\Example $example)
             return count($example);
         })->seeResultEquals(2);
     }
+
+    #[Examples(first: true, second: false)]
+    public function namedArguments(\CodeGuy $I, \Codeception\Example $example)
+    {
+        $I->assertSame(true, $example['first']);
+        $I->assertSame(false, $example['second']);
+    }
```

Comment:
Using named arguments was already be possible before this PR.

I took the chance to add a test case for them.

https://github.com/Codeception/codeception.github.com/pull/867 will add an example to the documentation.